companies/ingenico-eu.json

```
@@ -0,0 +1,36 @@
+{
+    "slug": "ingenico-eu",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "finance"
+    ],
+    "name": "Ingenico Group SA",
+    "runs": [
+        "Ingenico S.A.",
+        "Ingenico Payment Services GmbH",
+        "Ingenico Marketing Solutions GmbH",
+        "Ingenico Financial Solutions ",
+        "Ingenico e-Commerce Solutions",
+        "Ingenico Healthcare GmbH",
+        "Ingenico Terminals",
+        "Ingenico ePayments"
+    ],
+    "address": "28/32 boulevard de Grenelle\n75015 Paris\nFrankreich",
```

As this record is relevant in all countries, we have to fallback to the English country name here.

```suggestion
    "address": "28/32 boulevard de Grenelle\n75015 Paris\nFrance",
```